samples/java/src/main/java/com/example/Sample.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.example;
+
+import java.io.FileReader;
+import java.security.PrivateKey;
+import java.security.cert.X509Certificate;
+import java.time.LocalDateTime;
+import io.grpc.Channel;
+import io.grpc.netty.shaded.io.grpc.netty.GrpcSslContexts;
+import io.grpc.netty.shaded.io.grpc.netty.NettyChannelBuilder;
+import org.hyperledger.fabric.client.Contract;
+import org.hyperledger.fabric.client.Gateway;
+import org.hyperledger.fabric.client.Network;
+import org.hyperledger.fabric.client.identity.*;
+
+public class Sample
+{
+    private static final String mspID        = "Org1MSP";
+    private static final String cryptoPath   = "../../scenario/fixtures/crypto-material/crypto-config/peerOrganizations/org1.example.com";
```

Comment:
It would be nice to avoid `/` path separators. E.g. using something like this instead...
```
private static final Path cryptoPath = Paths.get("..", "..", "scenario", "fixtures", "crypto-material", "crypto-config", "peerOrganizations", "org1.example.com");
```